src/sentry/utils/snuba.py

```diff
@@ -198,7 +199,11 @@ def log_snuba_info(content):
     "timestamp": "timestamp",
     "trace": "trace_id",
     "transaction": "segment_name",
+    # `transaction.id` and `segment.id` is going to be replaced by `transaction.span_id` please do not use
+    # transaction.id is "wrong", its pointing to segment_id to return something for the transistion, but represents the
+    # txn event id(32 char uuid). EAP will no longer be storing this.
```

Comment:
might be good to prefix this with `DEPRECATED` so it stands out

Reply:
:+1: